src/sentry/web/frontend/debug/mail.py

```diff
@@ -211,11 +216,15 @@ def make_performance_event(project: Project, sample_name: str):
 
     with mock.patch.object(
         PerformanceNPlusOneGroupType, "noise_config", new=NoiseConfig(0, timedelta(minutes=1))
+    ), mock.patch.object(
+        PerformanceSlowDBQueryGroupType, "noise_config", new=NoiseConfig(0, timedelta(minutes=1))
     ):
         occurrence, group_info = process_event_and_issue_occurrence(
             occurrence_data,
             perf_data,
         )
+        produce_occurrence_to_kafka(payload_type=PayloadType.OCCURRENCE, occurrence=occurrence)
```

Comment:
@wedamija is this fine? Without it, an occurrence isn't produced at all and the page errors out because it can't find any events for the group

Reply:
idk the answer to this but shouldn't this slow db query issue stuff be inside a conditional like `if sample_name=whatever_slow_db_query_sample_name_is`?

Reply:
I don't think producing the occurrence has to be conditional, since this function is only called to create data for a performance issue, which is every sample that would come through here. On the other hand, I probably should change it to override the noise config only for the issue that's being loaded, eh?

Reply:
Ended up stubbing out the `noise_config` of the type of the occurrence in question, seems to work fine